perun/postprocess/regressogram/run.py

```diff
@@ -0,0 +1,76 @@
+"""
+Postprocessor module with non-parametric analysis using the regressogram method.
+"""
+import click
+
+import perun.logic.runner as runner
+import perun.postprocess.regressogram.methods as methods
+from perun.postprocess.regression_analysis.data_provider import data_provider_mapper
+from perun.postprocess.regression_analysis.tools import add_analysis_to_profile
+from perun.utils.cli_helpers import process_resource_key_param
+from perun.utils.helpers import PostprocessStatus, pass_profile
+
+__author__ = 'Simon Stupinsky'
+
+_DEFAULT_BINS_METHODS = 'doane'
+_DEFAULT_STATISTIC = 'mean'
+
+
+# TODO: The possibility of before postprocessing phase
+
+def postprocess(profile, **configuration):
+    """
+    Invoked from perun core, handles the postprocess actions
+
+    :param dict profile: the profile to analyze
+    :param configuration: the perun and options context
+    """
+    # Perform the non-parametric analysis using the regressogram method
+    analysis = methods.compute(data_provider_mapper(profile, **configuration), configuration)
+
+    # Store the results
+    profile = add_analysis_to_profile(profile, analysis)
+
+    # Return the profile after the execution of regressogram method
+    return PostprocessStatus.OK, '', {'profile': profile}
+
+
+# TODO: The possibility of after postprocessing phase
+
+@click.command()
+@click.option('--bins', '-b', required=False, default=_DEFAULT_BINS_METHODS,
```

Comment:
Suggestion: I'm thinking if it would be better to rename this to something longer, but I'm not sure at this moment to which. So lets keep this.